beetsplug/subsonicupdate.py

```diff
@@ -88,26 +90,35 @@ def __format_url():
                 context_path = ''
             url = "http://{}:{}{}".format(host, port, context_path)
 
-        return url + '/rest/startScan'
+        return url + '/rest/startScan.view'
 
     def start_scan(self):
         user = config['subsonic']['user'].as_str()
         url = self.__format_url()
-        salt, token = self.__create_token()
-
-        payload = {
-            'u': user,
-            't': token,
-            's': salt,
-            'v': '1.15.0',  # Subsonic 6.1 and newer.
-            'c': 'beets',
-            'f': 'json'
-        }
+        if config['subsonic']['user'] == 'token':
```

Comment:
This should be `config['subsonic']['auth']`, right?

Reply:
correction done thanks